workers/loc.api/sync/summary.by.asset/index.js

```diff
@@ -49,7 +49,7 @@ class SummaryByAsset {
     const auth = await this.authenticator
       .verifyRequestUser({ auth: args?.auth ?? {} })
     const end = args?.params?.end ?? Date.now()
-    const start = moment.utc(end)
+    const start = args?.params?.start ?? moment.utc(end)
```

Comment:
Not sure to understand this one. 
We add the start ?? (End + 30 days) 
Maybe we can split it to not be confusing.

Reply:
We add the start ?? (End `-` 30 days)